server/__tests__/index_test.js

```diff
@@ -0,0 +1,27 @@
+const request = require('supertest');
+const app = require('../index')
+
+describe('GET /api/products', () => {
+    test('responds with json and 200 status', function (done) {
+        request(app)
+            .get('/api/products/')
+            .expect('Content-Type', /json/)
+            .expect(200, done);
+    });
+})
+describe('PUT /api/products', () => {
+    test('responds with json and 200 status', function (done) {
+        request(app)
+            .put('/api/products/1')
```

Comment:
Since in the tests you're making real requests and - as I understand - the records are added to your DB - the DB will be full of records. Did you consider using a separate (not the one you want to use for your website) DB for your tests?

Reply:
I did consider this, but didn't follow it through. Thanks, I'll look into this again!